<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayBlogGUI</class>
 <widget class="QWidget" name="DisplayBlogGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="createBlog">
        <property name="text">
         <string>Create new blog</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshBlogs">
        <property name="text">
         <string>Refresh Blogs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="viewBlogButton">
        <property name="text">
         <string>View Blog</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
